### PR TITLE
Add/Extend Contracts for PM ImportOrder/GetImportOrderState/GetOrders endpoints

### PR DESCRIPTION
**Pull request by claudiutoderas10**
Add/Extend Contracts for PM ImportOrder/GetImportOrderState/GetOrders endpoints

### CODE REVIEW

File: Applications/ProductionManager/Contracts/Import/ImportOrderResult.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿using Newtonsoft.Json;
+
+namespace HomagConnect.ProductionManager.Contracts.Import;
+
+/// <summary>
+/// Order import operation result
+/// </summary>
+public class ImportOrderResult
```
Comment (HOMAG-Boris-Wehrle) [inline Applications/ProductionManager/Contracts/Import/ImportOrderResult.cs:8]: "Response" would fit better.

File: Applications/ProductionManager/Contracts/Import/Order.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        /// The name of the customer of this order
+        /// </summary>
+        [JsonProperty(Order = 3)]
+        public string CustomerName { get; set; } = string.Empty;
+        /// <summary>
+        /// The date the order was created at
+        /// </summary>
+
+        [JsonProperty(Order = 4)]
+        public DateTimeOffset OrderDate { get; set; }
```
Comment (HOMAG-Boris-Wehrle) [inline Applications/ProductionManager/Contracts/Import/Order.cs:33]: Please clarify with @joBueche whether to use DateTime or DateTimeOffset. All datetimes should be returned in the TimeZone which is set in the subscription.

File: Applications/ProductionManager/Contracts/Import/Order.cs [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        /// </summary>
+
+        [JsonProperty(Order = 5)]
+        public DateTimeOffset? DeliveryDate { get; set; }
+
+        /// <summary>
+        /// The state of the order. Possible states are:`New` / `ReadyForProduction` / `InProduction` / `Finished` 
+        /// </summary>       
+        [JsonProperty(Order = 4)]
+        public string State { get; set; } = string.Empty;
```
Comment (HOMAG-Boris-Wehrle) [inline Applications/ProductionManager/Contracts/Import/Order.cs:46]: Please replace with an enumeration.

File: Applications/ProductionManager/Samples/Orders/GetOrdersSamples.cs [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+{
+    /// <summary>
+    /// Sample class which shows how to get orders.
+    /// </summary>
+    public static class GetOrderSamples
+    {
+        /// <summary>
+        /// Gets the available orders for a customer.
+        /// </summary>
+        /// <param name="productionManager"></param>
```
Comment (HOMAG-Boris-Wehrle) [inline Applications/ProductionManager/Samples/Orders/GetOrdersSamples.cs:14]: Remove empty param documentation.

File: Applications/ProductionManager/Contracts/Import/Order.cs (hunk 3 above)
Comment (claudiutoderas10) [reply Applications/ProductionManager/Contracts/Import/Order.cs:46]: you mean something like this? 
image

Comment (HOMAG-Boris-Wehrle) [reply Applications/ProductionManager/Contracts/Import/Order.cs:46]: No, isn't there a enum "OrderState" or something like that? I just do not want to return a string in this case.

File: Applications/ProductionManager/Contracts/Import/ImportOrderResult.cs (hunk 1 above)
Comment (claudiutoderas10) [reply Applications/ProductionManager/Contracts/Import/ImportOrderResult.cs]: Updated

File: Applications/ProductionManager/Contracts/Import/Order.cs (hunk 3 above)
Comment (claudiutoderas10) [reply Applications/ProductionManager/Contracts/Import/Order.cs]: Updated

File: Applications/ProductionManager/Samples/Orders/GetOrdersSamples.cs (hunk 4 above)
Comment (claudiutoderas10) [reply Applications/ProductionManager/Samples/Orders/GetOrdersSamples.cs]: updated

**Review by HOMAG-Boris-Wehrle — APPROVED** [review]

File: Applications/ProductionManager/Contracts/Import/Order.cs (hunk 2 above)
Comment (joBueche) [reply Applications/ProductionManager/Contracts/Import/Order.cs:33]: I summarized my thoughts in a OneNote document. I will provide the link in internal chat

**Pull request merged**